pkg/after-install.sh

```diff
@@ -23,6 +23,18 @@ fi
 # Create log directory if it does not exist
 [[ -d $log_dir ]] || mkdir -p $log_dir && chown $user:$group $log_dir
 
+# if jsvc logs don't exist, create empty files
+if [[ ! -f $log_dir/wavefront-daemon.log ]]; then
+    > $log_dir/wavefront-daemon.log
+fi
+
+if [[ ! -f $log_dir/wavefront-error.log ]]; then
+    > $log_dir/wavefront-error.log
+fi
```

Comment:
probably fine to just use `touch` without an if-guard, but i guess that would modify the mtime and maybe you don't want that.